src/core/lib/slice/slice.cc

```diff
@@ -193,6 +216,24 @@ grpc_slice grpc_slice_from_copied_string(const char* source) {
   return grpc_slice_from_copied_buffer(source, strlen(source));
 }
 
+grpc_slice grpc_slice_from_moved_string(grpc_core::UniquePtr<char> p) {
```

Comment:
I assume that this was added here rather than as moved_buffer since there is currently no use for a moved_buffer; is that correct?